go-controller/pkg/node/ovspinning/ovspinning_linux_test.go

```diff
@@ -0,0 +1,187 @@
+//go:build linux
+// +build linux
+
+package ovspinning
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"runtime"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/sys/unix"
+	"k8s.io/klog/v2"
+)
+
+func TestAlignCPUAffinity(t *testing.T) {
+
+	ovsDBPid, ovsDBStop := mockOvsdbProcess(t)
+	defer ovsDBStop()
+
+	ovsVSwitchdPid, ovsVSwitchdStop := mockOvsVSwitchdProcess(t)
+	defer ovsVSwitchdStop()
+
+	defer setTickDuration(20 * time.Millisecond)()
+	defer mockFeatureEnableFile(t, "1")()
```

Comment:
why are we creating the file at the end?

Reply:
This syntax is a little bit tricky: 
`mockFeatureEnableFile(t, "1")` is called immediately, while the return value (which is a function as well) is called deferred. The same happens on line 30 for `setTickDuration()`

I can refactor this to a more verbose (and less cryptic) way, if needed.

Reply:
Aah, I missed that the other day. All good! :)